trunk/src/modules/paraview.F90

```diff
@@ -14,7 +14,10 @@ module paraview
    integer,parameter  :: PARAVIEW_IO = 22
    logical            :: PARAVIEW_DUMP_GEOM = .FALSE.
    logical            :: PARAVIEW_DUMP_ATTR = .FALSE.
-   logical            :: SECOND_ORDER_VIS   = .FALSE.
+   logical            :: SECOND_ORDER_VIS   = .TRUE.
+
+   ! Flag for switching on VTU format for output meshes and solution Fields. FALSE switches to XDMF.
+   logical            :: VIS_VTU = .TRUE.
```

Comment:
Is making this the default backward compatible with `paraview_driver` routines from `problems` that you didn't modify?

Reply:
Change implemented: XDMF is the default option as its compatible with all the problems.